code/modules/mob/living/carbon/life.dm

```diff
@@ -367,6 +368,10 @@
 		if(W.processes) // meh
 			W.handle_process()
 
+/mob/living/carbon/proc/handle_death()
+	for(var/datum/reagent/R in reagents.reagent_list)
+		R.on_mob_dead(src)
```

Comment:
```suggestion
	for(var/r in reagents.reagent_list)
		var/datum/reagent/chem
		chem.on_mob_dead(src)
```
This is supposed to happen several times, not just once on death, correct?

Reply:
Yes - though I just realised nothing in this PR uses this presently. Should I atomise this out?